### PR TITLE
Applied some spellchecker suggestions

### PR DESCRIPTION
**Pull request by nyurik**

### CODE REVIEW

**Comment by nyurik** [comment]
The CI is failing here until #1530

File: src/column.rs [hunk 1]
```diff
@@ -60,7 +60,7 @@ impl Statement<'_> {
     /// https://www.sqlite.org/c3ref/column_name.html
     /// > The returned string pointer is valid...
     ///
-    /// `column_name` reference can become invalid if `stmt` is reprepared
+    /// `column_name` reference can become invalid if `stmt` is re-prepared
```
Comment (gwenn) [inline src/column.rs:63]: https://sqlite.org/search?s=d&q=reprepared

File: src/hooks/mod.rs [hunk 2]
```diff
@@ -54,7 +54,7 @@ pub struct AuthContext<'c> {
     /// The database name, if applicable.
     pub database_name: Option<&'c str>,
 
-    /// The inner-most trigger or view responsible for the access attempt.
+    /// The innermost trigger or view responsible for the access attempt.
```
Comment (gwenn) [inline src/hooks/mod.rs:57]: https://sqlite.org/search?s=d&q=inner-most

Comment (nyurik) [reply src/hooks/mod.rs:57]: Are you implying that SQLite docs should be updated as well, or that they introduced a new keyword with a new meaning?  https://www.merriam-webster.com/dictionary/innermost -- is this the meaning?

**Comment by nyurik** [comment]
@gwenn I am happy to adjust my PR - do you want to just keep these as is despite being somewhat non-standard?  Need some feedback :)

**Comment by gwenn** [comment]
@nyurik Why not just use the same terms as the ones used by SQLite documentation ?
https://en.wiktionary.org/wiki/innermost
> equivalent to inner +‎ -most.

**Comment by nyurik** [comment]
ok, will revert

**Comment by gwenn** [comment]
Thanks.

**Pull request merged**